src/main/java/frc/robot/commands/groups/ScoreCube.java

```diff
@@ -1,35 +1,25 @@
 package frc.robot.commands.groups;
 
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
-import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
 import frc.robot.Constants;
-import frc.robot.commands.drive.SetDriveRotationTarget;
 import frc.robot.commands.floorArm.FloorMoveArmToPostion;
 import frc.robot.commands.intake.floor.FloorDrop;
-import frc.robot.subsystems.ArmSubsystem;
 import frc.robot.subsystems.DrivetrainSubsystem;
 import frc.robot.subsystems.FloorArmSubsystem;
 import frc.robot.subsystems.FloorIntake;
 
-public class RotateAndScoreCube extends SequentialCommandGroup {
+public class ScoreCube extends SequentialCommandGroup {
 
-  public RotateAndScoreCube(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, ArmSubsystem arm, FloorIntake intake) {
+  public ScoreCube(DrivetrainSubsystem drive, FloorArmSubsystem floorArm, FloorIntake intake) {
     addCommands(
-      new SetDriveRotationTarget(drive, 180),
-      new WaitUntilCommand(() -> Math.abs(drive.getRotation().getDegrees() - 180) < 5),
-      new SetDriveRotationTarget(drive), 
-
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_SHOOTING_POSITION),
       new WaitCommand(0.5),
       new FloorDrop(intake, Constants.FLOOR_INTAKE_FAST_SPEED).withTimeout(1),
       new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT_POSITION),
-
-      new SetDriveRotationTarget(drive, 0),
-      new WaitUntilCommand(() -> Math.abs(drive.getRotation().getDegrees()) < 5),
-      new SetDriveRotationTarget(drive), 
-      new WaitCommand(0.5)
-      
+      new WaitCommand(0.5),
+      new InstantCommand(drive::zeroRotation, drive)
```

Comment:
I am ok to try this out but I think we should not have to zero the rotation here if our path had the starting orientation represented via its initial holonomic angle. The challenge would be that this would then require different cone / cube paths but in my opinion is slightly less of a hack in terms of design.

I will approve and merge as it but something to keep in mind that in general it would be a best practice if our path defined our starting orientation.

Reply:
Testing has shown that you are corrected, latest changes reflect this.